### PR TITLE
Pulpcore 3.17 package bumps e - p

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

**Comment by zjhuntin** [comment]
[test rpm]

File: packages/python-pulp-deb/python-pulp-deb.spec [hunk 1]
```diff
@@ -5,7 +5,7 @@
 %global pypi_name pulp-deb
 
 Name:           %{?scl_prefix}python-%{pypi_name}
-Version:        2.16.1
+Version:        2.16.0
```
Comment (quba42) [inline packages/python-pulp-deb/python-pulp-deb.spec:8]: Why is this PR (I am assuming it is using some kind of automation) downgrading? At the time when this PR was opened pulp_deb `2.17.0` (the natural pulp_deb release to go with pulpcore 3.17) was already released. Is there anything I can help to make this upgrade process run more smoothly for pulp_deb?

Comment (Odilhao) [reply packages/python-pulp-deb/python-pulp-deb.spec:8]: @quba42 this PR was created to bump the dependencies for 3.17, we merged #368 and I'll close this one, I'm working in another PR right now to bump all the packages for 3.17, it's on my fork, here is the spec that will bump pulp-deb https://github.com/Odilhao/pulpcore-packaging/tree/rpm/3.17-pulpcore-release/packages/python-pulp-deb

**Comment by Odilhao** [comment]
Closing this PR, since #368 was merged and another PR will be created to bump to 3.17

File: packages/python-pulp-deb/python-pulp-deb.spec (hunk 1 above)
Comment (quba42) [reply packages/python-pulp-deb/python-pulp-deb.spec:8]: Thanks, that looks good then. I only commented to try and catch any possible mistakes early. I guess I don't really understand the process for bumping all the versions when it is time for a new pulpcore version. That being said, if ever there is a problem or question regarding pulp_deb in particular, please feel free to ping me.

**Pull request closed without merging**